Update action.yaml

mmaclean13 opened the pull request:
Updated code to use the 2.4 version of the VC-Utils action

mmaclean13 committed:
Update action.yaml

Updated code to use the 2.4 version of the VC-Utils action

nathangiusti commented on action.yaml:
```diff
@@ -12,7 +12,7 @@ runs:
     - name: Power BI VC Utils
-      uses: nathangiusti/Power-BI-VC-Utils@v2.3
+      uses: nathangiusti/Power-BI-VC-Utils@v2.4
       with:
```
This new VC utils takes in a separator argument right? Do we need to add that argument here?

mmaclean13 replied:
You're right, the 2.4 version takes that in

mmaclean13 replied:
Updated the file to remove the argument

nathangiusti replied:
So you need to do the opposite. 

You need to add the argument back into the tj actions

And then also pass the separator argument to our Power BI VC Utils action. 

The beauty of the composite action is that it doesn't matter what the separator is for the two actions, they just need to match. So instead of forcing the user to put those arguments in, we do it for them and we have arbitrarily chosen commas because commas generally don't appear in file paths. If someone committed a file path with a comma that would actually break stuff but nothing is perfect.

mmaclean13 committed:
Update action.yaml

Removed the separator argument as that is present in the v2.4 Power BI VC Utils action

mmaclean13 committed:
Update action.yaml

Added separator argument for the VC Utils action

mmaclean13 committed:
Update README.md

Updated the ReadMe to reflect the new version of the action which will be deployed with this pull request

mmaclean13 commented:
@nathangiusti - I've updated the action.yaml and the readme.md files as discussed

nathangiusti commented:
@mmaclean13 https://github.com/nathangiusti/PBIX-Deserializer/releases/tag/v1.3
Test that locally and then we can roll it out to the repos.
